ironfish/src/rpc/routes/chain/getBlock.ts

```diff
@@ -167,18 +154,14 @@ routes.register<typeof GetBlockRequestSchema, GetBlockResponse>(
     const main = await node.chain.isHeadChain(header)
     const confirmed = node.chain.head.sequence - header.sequence >= confirmations
 
+    const blockHeaderResponse = serializeRpcBlockHeader(header)
+
     request.end({
       block: {
-        graffiti: BufferUtils.toHuman(header.graffiti),
-        difficulty: header.target.toDifficulty().toString(),
-        hash: header.hash.toString('hex'),
-        previousBlockHash: header.previousBlockHash.toString('hex'),
-        sequence: Number(header.sequence),
+        ...blockHeaderResponse,
         size: getBlockSize(block),
-        timestamp: header.timestamp.valueOf(),
-        noteSize: header.noteSize,
-        noteCommitment: header.noteCommitment.toString('hex'),
-        transactions: transactions,
+        work: block.header.work.toString(),
```

Comment:
is `work` needed here? it isn't in the schema definition

Reply:
you're right, don't need it here. will remove